test/image/mocks/gl2d_fill_trace_filly.json

```diff
@@ -0,0 +1,52 @@
+{
+  "data": [
```

Comment:
Thanks for covering all these cases!

Now, WebGL image tests are expensive (especially on CI), so would you mind collecting all these cases in a single json mock using subplots?

Reply:
do you a example of subplots ?  @etpinard  

For #2765  i dont really understand why my patch resolve it .

Reply:
The easiest way to make subplots is with the `layout.grid` object - see eg the `stacked_area` [mock](https://github.com/plotly/plotly.js/blob/cfc720bd3c127ea0e206b4e3c2217b356ca92425/test/image/mocks/stacked_area.json#L81) and [image](https://github.com/plotly/plotly.js/blob/master/test/image/baselines/stacked_area.png)
Note in particular `"pattern": "independent"` which makes a separate x/y axis pair for each subplot, ie `xaxis/yaxis` in the first cell, `xaxis2/yaxis2` in the second (which you reference in the traces you want to show up there as `"xaxis": "x2", "yaxis": "y2"` and so on.

Reply:
@etpinard for #2765 ... does not solve more complex thing . example

https://codepen.io/erwanmas/pen/NLwXab

Reply:
only one mock with subplots